src/OrbitQt/TimeGraphLayoutWidgetTest.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QApplication>
+
+#include "TimeGraphLayoutWidget.h"
+
+TEST(TimeGraphLayoutWidgetTest, SetScale) {
+  orbit_qt::TimeGraphLayoutWidget widget{};
+  // There is not a lot of logic in the TimeGraphLayoutWidget, but we can at least test the SetScale
```

Comment:
It could be nice to test TimeGraphLayoutWidget::GetEventTrackHeightFromTid providing values for `all_threads_event_track_scale_` and `event_track_height_`.

Reply:
Good idea. Done.